platform/core.windows/src/org/netbeans/core/windows/services/NbPresenter.java

```diff
@@ -1278,8 +1288,9 @@ public void actionPerformed(ActionEvent e) {
     private class ButtonListener implements ActionListener, ComponentListener, PropertyChangeListener {
         ButtonListener() {}
         public void actionPerformed(ActionEvent evt) {
-            boolean isAqua = "Aqua".equals (UIManager.getLookAndFeel().getID()) || //NOI18N
-                            "true".equalsIgnoreCase (System.getProperty ("xtest.looks_as_mac"));
+            boolean useMacButtonOrder = "Aqua".equals (UIManager.getLookAndFeel().getID()) || //NOI18N
+                    UIManager.getLookAndFeel().getID().startsWith("FlatLaf") && Utilities.isMac() ||
+                    "true".equalsIgnoreCase (System.getProperty ("xtest.looks_as_mac"));
```

Comment:
/you forgot this usage i think. Could call the new utility method mentioned above.

Reply:
Oh yeah, forgot that one... will fix.

Reply:
Fixed the the other usage. That also allowed me to remove the "Aqua" check without breaking NbPresenterTest.